Javascript length property entry addition (#2905)
Codecademy/docs: content/javascript/concepts/storage/terms/length/length.md

---
Title: '.length'

> Description: 'The length property returns the number of items in an array or the number of characters in a string.'

Reviewer: ```suggestion
Description: 'Returns the number of items in an array or the number of characters in a string.'
```

Author: @yangc95 This change is now implemented. Please take a look. Thanks

Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Data'
  - 'Strings'
  - 'Arrays'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

The **`.length`** property is used on arrays and strings to quickly find out how many items are in an array or how many characters are in a string. This property is straightforward to use and plays a crucial role in JavaScript, making it simple for developers to work with the data.

## Syntax

```pseudo
myString.length
```

> > **Note**: The `.length` property is not a method but a property. It cannot be invoked with parentheses, like `.length()`. Instead, it can only be referenced as `.length`

Reviewer 2: ```suggestion
> **Note**: The `.length` property is not a method but a property. It cannot be invoked with parentheses, like `.length()`. Instead, it can only be referenced as `.length`.
```

---

> Title: 'Length'

Reviewer: ```suggestion
Title: '.length'
```

Description: 'The length property returns the number of items in an array or the number of characters in a string.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Data'
  - 'Strings'
  - 'Arrays'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

> The `.length` property is used with arrays and strings to quickly find out how many items are in an array or how many characters are in a string. This property is straightforward to use and plays a crucial role in Javascript, making it simple for developers to work with the data.

Reviewer: ```suggestion
The **`.length`** property is used on arrays and strings to quickly find out how many items are in an array or how many characters are in a string. This property is straightforward to use and plays a crucial role in JavaScript, making it simple for developers to work with the data.
```

## Syntax

```pseudo

> "Hello World!".length
> ["Apple", "Banana", "Cherry"].length

Reviewer: ```suggestion
myStringOrArray.length
```

```

> > **Note**: The `.length` property is not a method but a property. It cannot be invoked with parentheses, like `length()`. Instead, it can only be referenced as `.length`

Reviewer: ```suggestion
> **Note**: The `.length` property is not a method but a property. It cannot be invoked with parentheses, like `.length()`. Instead, it can only be referenced as `.length`
```

## Example

The `.length` property is applied to an array and a string, respectively, to quickly determine their sizes (number of elements for an array and number of characters for a string). This property provides a convenient and efficient way to work with data in JavaScript.

> ```javascript

Reviewer: ```suggestion
```js
```

const fruits = ['apple', 'banana', 'orange', 'grape'];
const numberOfFruits = fruits.length;
console.log(numberOfFruits);

const message = 'Hello, world!';
const messageLength = message.length;
console.log(messageLength);
```

This example results in the following output:

> ```javascript

Reviewer: ```suggestion
```shell
```

> Output: 4;
> Output: 13;

Reviewer: ```suggestion
4
13
```
